### PR TITLE
adds a Counter app

### PR DESCRIPTION
**Pull request by monsieurh**
request for comment
will be documented before merge

### CODE REVIEW

File: apps/counter/main.py [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+def get_char(counter):
+    return "|>" if counter.active else "||"
+
+
+def on_refresh_printing():
+    global i, o, counter
+    counter.on_refresh()
+    text_rows = ["{} {}".format(get_char(counter), round(counter.elapsed, 2)).center(o.cols)]
+    text_rows.extend([i.center(o.cols) for i in INSTRUCTIONS])
```
Comment (CRImier) [inline apps/counter/main.py]: While it's tempting to use ``i`` here, I think it's better to use something like ``str`` =)

File: apps/counter/main.py [hunk 2]
```diff
@@ -0,0 +1,99 @@
+# coding=utf-8
+from time import time
+
+from input.input import InputListener
```
Comment (CRImier) [inline apps/counter/main.py]: Does your IDE need this imported for autocomplete? Since this import is not actually used anywhere

Comment (monsieurh) [reply apps/counter/main.py]: Yes, it can be removed though. What do you think ?

**Comment by CRImier** [comment]
I suppose this PR's functionality is now provided by #23 =)

**Pull request merged**